dask_kubernetes/operator/tests/test_operator.py

```diff
@@ -35,7 +35,7 @@ async def cm():
             yield cluster_name
         finally:
             # Delete cluster resource
-            k8s_cluster.kubectl("delete", "-f", cluster_path)
+            k8s_cluster.kubectl("delete", "dsk", "--all")
```

Comment:
I think we should revert this. The test should only delete the cluster that it created.